src/test/java/POM/BasePage.java

```diff
@@ -0,0 +1,30 @@
+package POM;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.interactions.Actions;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+
+import java.time.Duration;
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.support.ui.WebDriverWait;
+
+public class BasePage {
+    protected WebDriver driver;
+    protected WebDriverWait wait;
+    Actions actions;
+    By firstPlaylist = By.cssSelector(".playlist:nth-child(3)");
+    By songTitle = By.cssSelector("[class='title']");
```

Comment:
Base Page \ Base Class — Implements the necessary methods for work with webdriver.

![image](https://user-images.githubusercontent.com/104761482/219848127-be1a40de-d1a3-46f5-a5e3-3e463a61b920.png)